packages/apps/human-app/frontend/src/components/layout/protected/layout.tsx

```diff
@@ -62,8 +67,26 @@ export function Layout() {
         backgroundColor: isMobile ? colorPalette.white : backgroundColor,
       }}
     >
-      <Navbar open={drawerOpen} setOpen={setDrawerOpen} />
+      <Navbar
+        open={drawerOpen || isMobileFilterDrawerOpen}
+        setOpen={setDrawerOpen}
+      />
       <DrawerNavigation drawerWidth={drawerWidth} open={drawerOpen} />
+      <AvaibleJobsDrawerMobile
```

Comment:
Move to jobs component, Full-screen dialogs - https://mui.com/material-ui/react-dialog/#full-screen-dialogs